packages/grafana-ui/src/components/uPlot/config/UPlotSeriesBuilder.ts

```diff
@@ -204,3 +213,104 @@ function getCanvasGradient(opts: AreaGradientOptions): (self: uPlot, seriesIdx:
     }
   };
 }
+
+function getCustomPointRenderer(shape: PointSymbol, color: string, size: number) {
+  let shapeSize: number;
+  let drawFn: (ctx: CanvasRenderingContext2D, cx: number, cy: number, height: number) => void;
+
+  switch (shape) {
+    case PointSymbol.Star:
+      shapeSize = size * 1.5;
+      drawFn = drawStar;
+      break;
+    case PointSymbol.Square:
+      shapeSize = size * 2;
+      drawFn = drawSquare;
+      break;
+    case PointSymbol.Triangle:
+      shapeSize = size * 3;
+      drawFn = drawTriangle;
+      break;
+    case PointSymbol.Marble:
+      shapeSize = size * 3;
+      drawFn = drawMarble;
+      break;
+    case PointSymbol.Cross:
+      shapeSize = size * 2;
+      drawFn = drawCross;
+      break;
```

Comment:
sizing is quick hacking here to make the shapes look sort of ok. Needs some love